src/main/webapp/help-filterLengthParameters.html

```diff
@@ -0,0 +1,5 @@
+<div>
+    <p>
+        The length of the pattern of the filter when the filter starts to work.
```

Comment:
I also think we need to improve this help documentation. Maybe a simple example, but I can take care of that after merging :-)